packages/origin-247-certificate/migrations/1646865380689-Add deploymentProperties.ts

```diff
@@ -0,0 +1,43 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddDeploymentProperties1646865380689 implements MigrationInterface {
+    name = 'AddDeploymentProperties1646865380689';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `CREATE TABLE "deployment_properties" ("registry" character varying NOT NULL, CONSTRAINT "PK_0b3f711854067e456735f0d25eb" PRIMARY KEY ("registry"))`
```

Comment:
I think this should be prefixed with standard prefix for 247-certificate tables (`certificate_` afair)